app/modules/trial_design/ui.R

```diff
@@ -1,240 +1,364 @@
 library(shiny)
+library(shinyjs)
+library(DT)
+library(escalation)
+library(trialr)
+library(ggplot2)
```

Comment:
If the package is not being used constantly, try to load each specific function by following our coding guideline (e.g. `packagename::function_name()` )